modules/build/src/main/scala/scala/build/Artifacts.scala

```diff
@@ -83,6 +85,25 @@ object Artifacts {
     logger: Logger
   ): Either[BuildException, Artifacts] = either {
 
+    addClang.map { _ =>
```

Comment:
That dangling call to `map` looks fishy:
```suggestion
    if (addClang.getOrElse(false)) {
```

Reply:
But when do we use that launcher? We should either fetch the launcher right before we use it, or keep the path to the fetched launcher in a field of `Artifacts`.

Reply:
(But maybe you just planned to do that in the final version of the PR?)